LinkedList.java

```diff
@@ -1,10 +1,18 @@
-import java.io.*; 
+import java.io.*;
+import java.net.http.HttpHeaders; 
+
+// Time Complexity : O(n)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : N/A - worked with different test cases, but only after proceeding failed build - what could be the problem here?
+// Any problem you faced while coding this : line 55 - question: return type is LinkedList, how would we return by head?

```

Comment:
`head` would be pointing to the first node of the LinkedList and it is being asked to return the first node of the list after appending.